newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -1,92 +1,275 @@
+                        )
+                      }
+                    />
+                    <Column expand noMargin>
+                      <ResponsiveLineStackLayout
+                        justifyContent="flex-end"
+                        noColumnMargin
+                      >
+                        <FlatButton
+                          label={<Trans>Access feedback</Trans>}
+                          onClick={() => {}}
+                          disabled={!game.publicWebBuildId}
                         />
-                      </>
-                    )}
-                    {game.publicWebBuildId && (
-                      <>
-                        <Spacer />
-                        <Chip
-                          size="small"
-                          label={
-                            game.discoverable ? (
-                              <Trans>Discoverable on Liluo.io</Trans>
-                            ) : (
-                              <Trans>Published on Liluo.io</Trans>
-                            )
-                          }
+                        <RaisedButton
+                          label={<Trans>Open in browser</Trans>}
+                          onClick={openGameUrl}
+                          primary
+                          disabled={!game.publicWebBuildId}
                         />
-                      </>
-                    )}
-                  </Line>
-                }
-              />
-              <CardActions>
-                <ResponsiveLineStackLayout
-                  expand
-                  noMargin
-                  justifyContent="flex-end"
-                >
-                  {game.publicWebBuildId && (
-                    <RaisedButton
-                      label={<Trans>Open in browser</Trans>}
-                      onClick={openGameUrl}
-                    />
-                  )}
-                  <RaisedButton
-                    label={<Trans>Manage game</Trans>}
-                    onClick={onOpenGameManager}
-                    primary
-                  />
-                </ResponsiveLineStackLayout>
-              </CardActions>
-            </Column>
-          </ResponsiveLineStackLayout>
-        </Card>
+                        <IconButton
+                          size="small"
+                          disabled={!game.publicWebBuildId}
+                          onClick={() => setShowShareDialog(true)}
+                          tooltip={t`Share`}
+                        >
+                          <ShareIcon />
+                        </IconButton>
+                      </ResponsiveLineStackLayout>
+                    </Column>
+                  </ResponsiveLineStackLayout>
+                  <ResponsiveLineStackLayout
+                    noMargin
+                    justifyContent="flex-start"
+                  >
+                    <Column expand alignItems="flex-start" noMargin>
+                      <Toggle
```

Comment:
Should those toggles open a small dialog warning the user what they're about to do so they can confirm?
As a newcomer or an experienced user, I'd feel more reassured by a 2 step process when making my game discoverable/open for feedback

Reply:
Dealing with it in #4063! :)